source.py

```diff
@@ -8,37 +8,77 @@
 # ## Topic
 # *What problem are you (or your stakeholder) trying to address?*
 # 📝 <!-- Answer Below -->
+# 
+# Some college football teams/programs are known for winning all the time. What is the difference between these teams and teams that do not.
 
 # ## Project Question
 # *What specific question are you seeking to answer with this project?*
 # *This is not the same as the questions you ask to limit the scope of the project.*
 # 📝 <!-- Answer Below -->
+# 
+# How do college football dynasties happen? What gives a college football team the edge to win? Are there certain factors that great teams have that help them win?
 
 # ## What would an answer look like?
 # *What is your hypothesized answer to your question?*
 # 📝 <!-- Answer Below -->
+# 
+# The answers could be anywhere from ability to recruit, if it was just straight ability to coach, could it be location and weather, and so on. As for graphics it will be charts that can describe different aspects of the game and give percentages and numbers back showing different things including win probability.
 
 # ## Data Sources
 # *What 3 data sources have you identified for this project?*
 # *How are you going to relate these datasets?*
 # 📝 <!-- Answer Below -->
+# 
+# I found a great website that allows you to query for data and download csv files. I have also found a dataset on Kaggle. I am also going to use Pro Football Focus' (PFF) data.
```

Comment:
It sounds like you have found good several reputable sources for your data. You may already know this, but it isn't spelled out here: how will you relate the different datasets to one another? Do each of them answer a specific aspect of your hypothesis, or will you need to combine information?